pkg/controller/v1alpha2/applicationconfiguration/update_trait_test.go

```diff
@@ -226,6 +227,8 @@ spec:
         - trait:
             apiVersion: example.com/v1
             kind: Bar
+            matadata:
+              name: example-trait
```

Comment:
why we need a name here?

Reply:
If no fixed trait name, I have to get the trait name from AppConfig status (need more code) or use the implicitly generated trait name by the controller.